content/release-notes/2022-10-21.md

```diff
@@ -0,0 +1,12 @@
+---
+label: 'Storage'
+---
+
+### What's new
+
+
+- Compute: Fixed an issue that prevented creating a database when the specified database name included trailing spaces.
+- Pageserver: Fixed an `INSERT ... ON CONFLICT` handling issue for speculative Write-Ahead Log (WAL) record inserts. Under certain load conditions, records added with `INSERT ... ON CONFLICT` could be replayed incorrectly.
+- Pageserver: Fixed a Free Space Map (FSM) and Visibility Map (VM) page reconstruction issue that caused compute nodes to start slowly under certain workloads.
+- Pageserver: Fixed a garbage collection (GC) issue that could lead to a database failure under high load.
+- Pageserver: Improve query performance for larger databases.
```

Comment:
@lubennikovaav I missed this one. Is there more context for "Improve query performance for larger databases."?

Reply:
Revised.